iohk-monitoring/src/Cardano/BM/Data/LogItem.lhs

```diff
@@ -128,15 +126,15 @@ mkLOMeta sev priv =
   where
     cleantid threadid = do
         let prefixText = "ThreadId "
-            condStripPrefix s = maybe s id $ stripPrefix prefixText s
+            condStripPrefix s = fromMaybe s $ stripPrefix prefixText s
         condStripPrefix $ (pack . show) threadid
 
 \end{code}
 
 Convert a timestamp to ns since epoch:\label{code:utc2ns}\index{utc2ns}
 \begin{code}
 utc2ns :: UTCTime -> Word64
-utc2ns utctime = fromInteger $ round $ 1000 * 1000 * 1000 * (utcTimeToPOSIXSeconds utctime)
+utc2ns utctime = fromInteger $ round $ 1000 * 1000 * 1000 * utcTimeToPOSIXSeconds utctime
```

Comment:
these braces support readability for humans. hlint can be too aggressive sometimes.

Reply:
https://typeclasses.com/ghc/numeric-underscores

Reply:
Done.